src/test/resources/unrc/dose/Bono.feature

```diff
@@ -0,0 +1,91 @@
+
+Feature: test based challenge equals
+     Scenario Outline: User solves an unsolved test based challenge.
+Given that user <user> is already logged on 
```

Comment:
Here <user> seems to describe a placeholder, but the `examples` section where the placeholder's values are defined is missing.